### ISSUE TITLE
Outlook Add-in problem

### ISSUE THREAD

**Issue by samuel0711** [browserify/browserify#1942]
Hello.

I'm doing a Outlook Add-in and found the Browserify module on my search to use the require to the 'mssql' NPM module. My job is to do a selectbox with some elements from a SQL Server DB.
Here is my taskpane.html where i create the selectbox:
![image](https://user-images.githubusercontent.com/20281744/73208164-b9335a80-4124-11ea-9de0-dfa5da817ee8.png)

*The <script src="bundle.js"></script> is where i do input the browserified code in the HTML.
**The doH() is a simples function to return an array with some strings in it.
![image](https://user-images.githubusercontent.com/20281744/73208864-efbda500-4125-11ea-9681-3567c76370e6.png)

Here is my testing_DB.js code:
![image](https://user-images.githubusercontent.com/20281744/73208516-5a221580-4125-11ea-840b-15f9de79f0ea.png)

*The acess to the DB is fine. I've tested many times with success.

The folder's path is like this:

**_-My Office Add-in
  ...
  -src
    ...
    -taskpane
      -bundle.js(the browserified code generated)
      -taskpane.html
      -testing_DB.js_**

Basically, i tried to do : **browserify testing_DB.js --standalone myBundle > bundle.js**
Where testing_DB.js is where i do require the 'mssql' module, and tried to insert the elements from the DB into the innerHTML of the selectbox.

My problem is simple. I don't think the browserify worked.

I put some document.writeln() in the testing_DB.js file, and don't have any answers from it.

This is how looks like when i click on the dropdown button:
![image](https://user-images.githubusercontent.com/20281744/73209673-8343a580-4127-11ea-984b-2122d413a032.png)

I've already tried many StackOverflow's threads help, but none of them worked.

Can u guys please help me?

If i could just fill the selectbox with an item from the DB is already something.

Thanks for your attention!

**Comment by samuel0711** [comment]
The tree of folders looks pretty bad in the issue, here's a print of it:
![image](https://user-images.githubusercontent.com/20281744/73210557-4f697f80-4129-11ea-9b04-e09761eacf31.png)

**Comment by goto-bus-stop** [comment]
mssql relies on node.js's ability to make raw tcp connections to a database. That is not something you can do in the browser, and probably not in outlook, at least not in the way node.js does it. You definitely don't _want_ to do it in the browser, because that would be wildly insecure :smile: 
I've never heard of outlook addins before so I'm not sure how much I can help. Is the goal to connect to a database on the user's local computer? You'd need to look into the APIs that outlook provides.
If the goal is to connect to a central database that you run elsewhere, you almost certainly need to write some backend HTTP APIs that do the database interactions, and use those HTTP endpoints in your addin code.

**Comment by samuel0711** [comment]
The Add-in is initialized via npm run dev-server, and is essential to do this way, because later i will use some e-mails information. Otherwise, doing the backend HTTP API would be the best case...
I've heard about being insecure, but the application will be used internally. So just 1 user will be using it. Still some problem to use it?
The goal is to connect to DB, do a query, and return a recordset to fill a combobox with some records elements.
